schemas/error-schema.json

```diff
@@ -141,4 +163,4 @@
       }
     }
   ]
-}
+}
```

Comment:
there is no lint check in this repository for now, it could worth check your IDE settings to ensure that you are not removing EOF new lines